src/main/java/ca/yorku/cse2311/tab2pdf/ui/listener/AboutListener.java

```diff
@@ -14,27 +17,79 @@
  * @author Glib Sitiugin, Anton Sitkovets, Marco Cirillo
  * @since 2015-03-19
  */
- public class AboutListener implements ActionListener {
+ public class AboutListener extends JPanel implements ActionListener{
 
     private final MainJFrame window;
 
+    JLabel InfoLabel,brodyPicLabel, brodyInfoLabel, glibPicLabel, glibInfoLabel, picLabel;
+
+    String deepName = "Deep Patel";
+    String deepEmail = "deep0410@my.yorku.ca";
+    String marcoName = "Marco Pietro Cirillo";
+    String marcoEmail = "cirillom@my.yorku.ca";
+    String antonName = "Anton Sitkovets";
+    String antonEmail = "antosi@my.yorku.ca";
+    String brodyName = "Brody Atto";
+    String brodyEmail = "brodyatto@gmail.com";
+    String varshaName = "Varsha Raghav";
+    String varshaEmail = "varsha_raghav@hotmail.com";
+    String glibName = "Glib Sitiguin";
+    String glibEmail = "situgin@yorku.ca";
+
     /**
      * @param window    the window we are working with
      */
-    public AboutListener(MainJFrame window) {
+    public AboutListener (MainJFrame window) {
+
+       super();
+
+       this.window = window;
+
+        ImageIcon deep = null;
```

Comment:
Inconsistent spacing starts here, make sure to run a code formatter before committing to reduce the likelihood of things like this